### PR TITLE
feat(chromium): roll to Chromium 92.0.4512.0 (r884014)

### PR DESCRIPTION
**Pull request by jschfflr**

### CODE REVIEW

**Review by mathiasbynens — APPROVED** [review]

File: versions.js [hunk 1]
```diff
@@ -17,6 +17,7 @@
 const versionsPerRelease = new Map([
   // This is a mapping from Chromium version => Puppeteer version.
   // In Chromium roll patches, use 'NEXT' for the Puppeteer version.
+  ['92.0.4512.0', 'v9.1.1'],
```
Comment (mathiasbynens) [inline versions.js:20]: This should have said `'NEXT'` per the comment above (sorry for missing this). Could you please patch this before cutting the release?

Comment (jschfflr) [reply versions.js:20]: As a separate cl or in this one?

Comment (jschfflr) [reply versions.js:20]: Ah, sorry. Got confused by the patches

**Review by splawless — COMMENTED** [review]
**_[image](https://github.com/puppeteer/puppeteer/pull/7288/files)_**

**Pull request merged**